client/lib/user/user.d.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * External Dependencies
+ */
+import { EventEmitter } from 'events';
+import { Language } from '../../languages';
+import { GravatarOptions } from '../../gravatar/types';
+import { URL } from '../../types';
+
+type JSONSerializable = boolean | string | number;
```

Comment:
this one I made up for the concept. it belongs in `client/types.ts` as a proper type signature - anything which can be JSON serialized (or in this case, unserialized)

<img width="400" alt="Screen Shot 2020-02-26 at 1 11 48 AM" src="https://user-images.githubusercontent.com/5431237/75324839-034b5100-5835-11ea-952f-fe4e9546e148.png">

(just the type - I added the `const` values to show that it's flagging invalid JSON-able data types)

Reply:
Ahh ok I see what you mean